src/installation/index.md

```diff
@@ -120,41 +132,73 @@ This will install all the necessary tools to develop Rust applications for all s
 [esp-rs/rust-build]: https://github.com/esp-rs/rust-build
 [Installing from Source section of the esp-rs/rust repository]: https://github.com/esp-rs/rust#installing-from-source
 
```

Comment:
@SergioGasquez 

Other installation methods refer to the instructions in `espup` and `esp-rs/rust` that I didn't find. Can you please check? Should we update the links here?